takhojin/controllers/userController.js

```diff
@@ -15,6 +15,22 @@ const signUp = async (req, res) => {
   }
 };
 
+const login = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+
+    if (!email || !password) {
+      return res.status(400).json({ message: "KEY_ERROR" });
+    }
+    const token = await userService.login(email, password);
+    console.log("token" + "" + token);
+    return res.status(201).json({ accessToken: token });
```

Comment:
```suggestion
    const accessToken = await userService.login(email, password);
    return res.status(200).json({ accessToken });
```
1. 로그인 처리에 맞는 status code를 사용할 수 있도록 해주세요.
2. javascript object에서는 property name과 value의 변수 name이 같다면 위와 같이 간단하게 작성할 수 있고 보다 깔끔한 코드를 작성하는데 도움됩니다 :)